src/plugins/path/README.md

```diff
@@ -26,7 +27,7 @@ relative files, it is not enough to look at the first character if it is
 a `/`, because remote file systems and some special names are valid, too.
 
 If `check/permission = [permission], [user]` is also present it will check for the correct permissions
-of the file/directory. `check/permission = rw, tomcat` for example will check if the user `tomcat` has read and 
+of the file/directory. `check/permission = rw` and `check/permission/user = tomcat` for example will check if the user `tomcat` has read and 
```

Comment:
There is still no explanation of check/permission, check/permission/user. It is good to have an example but there should also be an explanation without example.